### PR TITLE
Fix publish steps

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Semi-automate the publish steps (still need to enter credentials for git).

**Purpose**
Reduce human errors.

TODO: Consider adding this to a bash or python script. Since it's basically only used by me now this will do.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
Considering how many steps needed for publishing adapter, there is no wonder mistakes have been made ;)

File: README.md [hunk 1]
```diff
@@ -46,3 +46,7 @@ Include the file that suits your need in your project.
 Detailed information on developing in the [webrtc](https://github.com/webrtc) github repo can be found in the [WebRTC GitHub repo developer's guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit?pli=1#heading=h.e3366rrgmkdk).
 
 Head over to [test/README.md](https://github.com/webrtc/samples/blob/gh-pages/test/README.md) and get started developing.
+
+## Publish a new version ##
+Just run `npm version patch -m 'bump to %s'` and type in your password lots of times (setting up credential caching is probably a good idea).
+Currently only tested on Linux, not sure about MAC but will definitely not work on Windows.
```
Comment (samdutton) [inline README.md]: Mac

File: README.md [hunk 2]
```diff
@@ -46,3 +46,7 @@ Include the file that suits your need in your project.
 Detailed information on developing in the [webrtc](https://github.com/webrtc) github repo can be found in the [WebRTC GitHub repo developer's guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit?pli=1#heading=h.e3366rrgmkdk).
 
 Head over to [test/README.md](https://github.com/webrtc/samples/blob/gh-pages/test/README.md) and get started developing.
+
+## Publish a new version ##
+Just run `npm version patch -m 'bump to %s'` and type in your password lots of times (setting up credential caching is probably a good idea).
```
Comment (samdutton) [inline README.md]: Maybe add a line space after.

**Comment by samdutton** [comment]
LGTM though I haven't tried running the code.

**Pull request merged**